manifests/v1beta1/katib-controller/katib-controller.yaml

```diff
@@ -18,6 +18,10 @@ spec:
         prometheus.io/scrape: "true"
     spec:
       serviceAccountName: katib-controller
+      initContainers:
+        - name: cert-generator
+          image: docker.io/kubeflowkatib/cert-generator
+          imagePullPolicy: Always
```

Comment:
```suggestion
          imagePullPolicy: Always
          env:
            - name: KATIB_CORE_NAMESPACE
              valueFrom:
                fieldRef:
                  fieldPath: metadata.namespace          
```

Reply:
```suggestion
          imagePullPolicy: Always
          env:
            - name: KATIB_CORE_NAMESPACE
              valueFrom:
                fieldRef:
                  fieldPath: metadata.namespace
```

@andreyvelich 

Sorry, when I added ```env``` fields before, I seem to have added an unnecessary margin. Could you please erase it?

Reply:
Which margin do you mean ? I think you did it correct, compare this:
https://github.com/andreyvelich/katib/blob/issue-1405-disable-dynamic-webhooks/manifests/v1beta1/katib-controller/katib-controller.yaml#L25-L29 and
https://github.com/andreyvelich/katib/blob/issue-1405-disable-dynamic-webhooks/manifests/v1beta1/katib-controller/katib-controller.yaml#L49-L53

Reply:
@andreyvelich 

There are 10 spaces at the end of the line.

https://github.com/andreyvelich/katib/blob/issue-1405-disable-dynamic-webhooks/manifests/v1beta1/katib-controller/katib-controller.yaml#L29

Reply:
Oh makes sense, thank you!